controllers/admin/AdminSearchConfController.php

```diff
@@ -371,8 +371,8 @@ public function renderForm()
 
     public function processSave()
     {
-        $search = strval(Tools::getValue('search'));
-        $string = strval(Tools::getValue('alias'));
+        $search = (string) (Tools::getValue('search'));
+        $string = (string) (Tools::getValue('alias'));
```

Comment:
This is also true is another places of this PR, some parentheses can be safely removed.
```suggestion
        $string = (string) Tools::getValue('alias');
```